packages/nameguard-sdk/src/utils.ts

```diff
@@ -25,3 +27,7 @@ const keccak256Regex = /^(?:0x)?[0-9a-f]{64}$/i;
 export function isKeccak256Hash(hash: string) {
   return keccak256Regex.test(hash);
 }
+
+export function getNameGuardURLForENSname(ensName: ENSName): string {
+  return `https://nameguard.io/inspect/${encodeURIComponent(ensName.name)}`;
```

Comment:
I'm not a fan of this utility in general living inside the SDK, but if we are to have it, we should make sure the SDK generates this using the `endpoint` we pass the constructor:

```ts
class NameGuard {
  private endpoint: URL;
  private network: Network;
  private abortController: AbortController;

  constructor({ endpoint = DEFAULT_ENDPOINT, network = DEFAULT_NETWORK } = {}) {
    this.endpoint = new URL(endpoint);
    this.network = network;
    this.abortController = new AbortController();
  }
```

I would move this function to a public function on the `NameGuard` class that takes in a name to generate the URL.

It's good practice we do this so if we ever want to run Nameguard locally or on another URL, we have fewer constants to change.

cc @lightwalker-eth @FrancoAguzzi

Reply:
@notrab thanks for the feedback! Very helpful towards clean coding 

@lightwalker-eth I will wait for your thoughts around how this implementation should look like before doing code updates

Reply:
I wrote a big reply but then GitHub lost it right before I submitted it due to some bug 😞 Will write a summarized version as round 2.

@notrab I agree with your feedback that this logic probably doesn't belong in the SDK. The SDK is for API-level concerns. The goal that we're discussing now is different than the `endpoint` param passed to a `NameGuard` instance. That `endpoint` param is for API-level requests, but the idea we're talking about is for URLs that end users would use to view a NameGuard report.

@notrab @FrancoAguzzi  Therefore perhaps we should move this logic into the react SDK instead somewhere? Appreciate advice on where we might move that.

Reply:
NameGuard React package seems a good home to this utility

Reply:
This is **100%** related to web and it will be used by front-end logics only

Reply:
@FrancoAguzzi @notrab Ok cool! Let's proceed with moving this logic then :+1:

Reply:
Indeed it seems interesting to provide this to devs integrating NameGuard reports... Otherwise it could live inside `apps/nameguard.io`... But devs might be interested in reusing it... At least this is what I see now, could be changed in the future if no _integrators_ adoption, to the `nameguard.io` "namehash-labs-only" folder